### ISSUE TITLE
[master] infoWindow does not update content based on $scope variables

### ISSUE THREAD

**Issue by franklindner** [angular-ui/angular-google-maps#354]
I want to show the distance and duration between two markers in an infoWindow. Those variables are getting updated every 10 seconds as a result of a request and changing marker positions. I can print out $scope.distance.duration it is getting updated nicely but the infoWindow never shows them.

HTML:
 <window show="map.infoWindow.show" coords="map.infoWindow.coords" isIconVisibleOnClick="false" options="map.infoWindow.options">
        <p>Distance: {{ distanceMatrix.distance.text }} | Duration: {{ distanceMatrix.duration.text }}</p>
      </window>

Could this be an issue or am I just doing it wrong?

**Comment by knitzie** [comment]
Hi,
I just found this issue and updated the bower package on my project to solve my issue. 
Now I've got a strange behaviour: After first click on the marker, the window opens empty. On second click, the content is available.
I forked a fiddle and add a random variable (windowOptions.show) to the html content:
http://jsfiddle.net/martgvd0/

FYI:
In my case, I want to update the window content with the markers data (like the address).

Does anyone else got this issue too?
